vitest.config.mjs

```diff
@@ -6,16 +6,22 @@ import { vanillaExtractPlugin } from '@vanilla-extract/vite-plugin'
 // https://vitejs.dev/config/
 export default defineConfig({
     plugins: [react(), tsconfigPaths(), vanillaExtractPlugin()],
+    setupFiles: './vitest.setup.ts',
```

Comment:
what I ran into while trying to DIY: path should be `./tests/vitest.setup.ts'` and `setupFiles` didn't do much unless it was under `test`. did this work for you??